vue/components/ui/organisms/details-expandable/details-expandable.stories.js

```diff
@@ -0,0 +1,71 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs } from "@storybook/addon-knobs";
+
+storiesOf("Components/Organisms/Details Expandable", module)
+    .addDecorator(withKnobs)
+    .add("Details Expandable", () => ({
+        props: {
+            data: {
+                type: Object,
+                default: () => ({
+                    first_row: {
+                        field_a: "something",
+                        field_b: "something else",
+                        field_object: {
+                            property_a: "property a",
+                            property_b: "property b"
+                        }
+                    },
+                    second_row: {
+                        field_c: "yet something",
+                        field_d: "yet something else",
+                        field_object: {
+                            property_c: "property c",
+                            property_d: "property d"
+                        }
+                    }
+                })
+            },
+            labelFontSize: {
+                default: 14
+            },
+            discardButtonText: {
+                default: "Discard changes or not..."
+            },
+            saveButtonText: {
+                default: "Save changes now!"
+            }
+        },
+        methods: {
+            onDiscardClick() {
+                console.log("onDiscardClick");
+            },
+            onSaveClick() {
+                console.log("onSaveClick");
+            }
+        },
+        template: `
+            <details-expandable-ripe
+                v-bind:data="data"
+                v-bind:label-font-size="labelFontSize"
+                v-bind:style="'margin-top: 25px'"
+                v-bind:discard-button-text="discardButtonText"
+                v-bind:save-button-text="saveButtonText"
+                v-on:discard:click="onDiscardClick"
+                v-on:save:click="onSaveClick"
+            >
+                <template v-slot:first_row>
+                    Customized Customer
+                </template>
+                <template v-slot:first_row-label-field_name>
+                    name
+                </template>
+                <template v-slot:first_row-field_object>
+                    <input-ripe v-bind:style="'width: 50%'" v-bind:value="data.first_row.field_object.property_a" />
+                </template>
+                <template v-slot:second_row-field_object>
+                    <input-ripe v-bind:style="'width: 50%'" v-bind:value="data.second_row.field_object.property_c" />
+                </template>

```

Comment:
Don't directly pass text to a slot, wrap it in a div.

If you want to only change the values use a prop instead. For example, if you only want to change the text in the header, instead of passing text via a slot, create a prop that allows to change the header text.

If it's a field that is dynamically generated, make the prop that allows to generate that dynamic structure accept some kind of field that would change the header. For example, in this case I think it would be to also allow some kind of field to `data` that would handle that change.
Check other components for reference. For example: https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/atoms/dropdown/dropdown.vue

Reply:
what if we want to change only certain inputs and not all?